Feat spanish translation team.md page (#759)
reactjs/es.react.dev: src/content/community/team.md

---
title: "Conoce al equipo"
---

<Intro>

El desarrollo de React está dirigido por un equipo dedicado que trabaja a tiempo completo en Meta. También recibe contribuciones de personas de todo el mundo.

</Intro>

## Equipo principal de React {/*react-core*/}

Los miembros del equipo de React Core trabajan a tiempo completo en las API de componentes principales, el motor que impulsa React DOM y React Native, React DevTools y el sitio web de documentación de React.

Los miembros actuales del equipo de React se enumeran en orden alfabético a continuación.
Engineer at Meta
<TeamMember name="Andrew Clark" permalink="andrew-clark" photo="/images/team/acdlite.jpg" github="acdlite" twitter="acdlite" title="Ingeniero en Vercel">

>     Andrew comenzó con el desarrollo web creando sitios con WordPress, y y finalmente se engañó a sí mismo para hacer JavaScript. Su pasatiempo favorito es el karaoke. Andrew es un villano una princesa de Disney, según el día.

Reviewer: ```suggestion
    Andrew comenzó con el desarrollo web creando sitios con WordPress, y y finalmente se engañó a sí mismo para hacer JavaScript. Su pasatiempo favorito es el karaoke. Andrew es un villano o una princesa de Disney, según el día.
```

<TeamMember name="Andrey Lunyov" permalink="andrey-lunyov" photo="/images/team/andrey-lunyov.jpg" github="alunyov" twitter="alunyov" title="Ingeniero en Meta">
    Andrey comenzó su carrera como diseñador y luego pasó gradualmente al desarrollo web. Después de unirse al equipo de React Data en Meta, trabajó para agregar un compilador de JavaScript incremental a Relay y, más tarde, trabajó para eliminar el mismo compilador de Relay. Fuera del trabajo, a Andrey le gusta tocar música y practicar varios deportes.
</TeamMember>

<TeamMember name="Dan Abramov" permalink="dan-abramov" photo="/images/team/gaearon.jpg" github="gaearon" twitter="dan_abramov" title="Ingeniero en Meta">
    Dan comenzó a programar después de que accidentalmente descubrió Visual Basic dentro de Microsoft PowerPoint. Ha encontrado su verdadera vocación al convertir los tweets de [Sebastian](#sebastian-markbåge) en extensas publicaciones de blog. Dan gana ocasionalmente en Fortnite escondiéndose en un arbusto hasta que termina el juego.
</TeamMember>

<TeamMember name="Dave McCabe" permalink="dave-mccabe" photo="/images/team/dave-mccabe.jpg" github="davidmccabe" twitter="mcc_abe" title="Ingeniero en Meta">

>     Ingeniero de oficio y amante de la naturaleza de corazón, David ha sido durante mucho tiempo un innovador en el campo de la programación mientras toma el sol. Además de sorprender a sus colegas con fondos únicos al aire libre en las videollamadas, le gusta tocar la guitarra (en prados iluminados por el sol, por supuesto) y las artes marciales (todavía en el interior, tengo que trabajar en eso).

Reviewer: ```suggestion
    Ingeniero de profesión y amante de la naturaleza de corazón, David lleva mucho tiempo innovando en el campo de la programación mientras se toma el sol. Además de sorprender a sus colegas con fondos de videollamada únicos al aire libre, le gusta tocar la guitarra (en praderas iluminadas por el sol, por supuesto) y las artes marciales (todavía en interiores, debe trabajar en ello).
```

</TeamMember>

<TeamMember name="Eli White" permalink="eli-white" photo="/images/team/eli-white.jpg" github="TheSavior" twitter="Eli_White" title="Gerente de ingeniería en Meta">
    Eli comenzó a programar después de que lo suspendieran de la escuela secundaria por piratería informática. Ha estado trabajando en React y React Native desde 2017. Le gusta comer golosinas, especialmente helados y tartas de manzana. Puedes encontrar a Eli probando actividades extravagantes como parkour, paracaidismo bajo techo y actividades aéreas.
</TeamMember>

<TeamMember name="Jason Bonta" permalink="jason-bonta" photo="/images/team/jasonbonta.jpg" title="Gerente de ingeniería en Meta">

> A Jason le gusta recibir grandes volúmenes de paquetes de Amazon en la oficina para poder construir fuertes. A pesar de que a veces se separa literalmente de su equipo y no comprende cómo funcionan los bucles for-of, lo apreciamos por las cualidades únicas que aporta a su trabajo.

Reviewer: ```suggestion
    A Jason le gusta recibir grandes cantidades de paquetes de Amazon en la oficina para poder construir fuertes. A pesar de que a veces se aísla literalmente de su equipo y no entiende cómo funcionan los bucles de for-of, le apreciamos por las cualidades únicas que aporta a su trabajo.
```

</TeamMember>

<TeamMember name="Joe Savona" permalink="joe-savona" photo="/images/team/joe.jpg" github="josephsavona" twitter="en_JS" title="Ingeniero en Meta">

>     Joe planeaba especializarse en matemáticas y filosofía, pero se dedicó a la informática después de escribir simulaciones de física en Matlab. Antes de React, trabajó en Relay, RSocket.js y el lenguaje de programación Skip. Si bien no está construyendo algún tipo de sistema reactivo, disfruta correr, estudiar japonés y pasar tiempo con su familia.

Reviewer: ```suggestion
    Joe planeaba especializarse en matemáticas y filosofía, pero se metió en la informática después de escribir simulaciones de física en Matlab. Antes de React, trabajó en Relay, RSocket.js y el lenguaje de programación Skip. Cuando no está construyendo algún tipo de sistema reactivo, le gusta correr, estudiar japonés y pasar tiempo con su familia.
```

</TeamMember>

<TeamMember name="Josh Story" permalink="josh-story" photo="/images/team/josh.jpg" github="gnoff" twitter="joshcstory" title="Ingeniero en Vercel">

>     Josh se especializó en Matemáticas y descubrió la programación mientras estaba en la universidad. Su primer trabajo de desarrollador profesional fue programar cálculos de tarifas de seguros en Microsoft Excel, el modelo de Programación Reactiva, que debe ser la razón por la que ahora trabaja en React. Entre ese tiempo, Josh ha sido IC, Gerente y Ejecutivo en algunas nuevas empresas. fuera del trabajo le gusta empujar sus límites con la cocina.

Reviewer: ```suggestion
    Josh se especializó en Matemáticas y descubrió la programación en la universidad. Su primer trabajo como desarrollador profesional fue programar cálculos de tarifas de seguros en Microsoft Excel, el parangón de la programación reactiva, que debe ser la razón por la que ahora trabaja en React. En el medio de ese tiempo Josh ha sido un IC, Gerente y Ejecutivo en algunas nuevas empresas. fuera del trabajo le gusta empujar sus límites con la cocina.
```

>     Josh se especializó en Matemáticas y descubrió la programación mientras estaba en la universidad. Su primer trabajo de desarrollador profesional fue programar cálculos de tarifas de seguros en Microsoft Excel, el modelo de Programación Reactiva, que debe ser la razón por la que ahora trabaja en React. Entre ese tiempo, Josh ha sido IC, Gerente y Ejecutivo en algunas nuevas empresas. fuera del trabajo le gusta empujar sus límites con la cocina.

Reviewer: ```suggestion
    Josh se especializó en Matemáticas y descubrió la programación en la universidad. Su primer trabajo como desarrollador profesional fue programar cálculos de tarifas de seguros en Microsoft Excel, el parangón de la Programación Reactiva, que debe ser la razón por la que ahora trabaja en React. En el medio de ese tiempo Josh ha sido un IC, Gerente y Ejecutivo en algunas nuevas empresas. fuera del trabajo le gusta empujar sus límites con la cocina.
```

</TeamMember>

<TeamMember name="Kathryn Middleton" permalink="kathryn-middleton" photo="/images/team/kathryn-middleton.jpg" github="kmiddleton14" twitter="kmiddleton14" title="Gerente de ingeniería en Meta">
    Kathryn descubrió inicialmente el desarrollo web cuando quería que su página de myspace se viera genial. Terminó especializándose en Informática y rápidamente se convirtió en una gran admiradora de las funciones de creación de React en el equipo de Instagram.com. Fuera del trabajo, le encanta jugar al ping pong, dar clases de spinning e ir a comprar plantas.
</TeamMember>

<TeamMember name="Lauren Tan" permalink="lauren-tan" photo="/images/team/lauren.jpg" github="poteto" twitter="potetotes" personal="no.lol" title="Ingeniera en Meta">

>     La carrera de programación de Lauren alcanzó su punto máximo cuando descubrió por primera vez la etiqueta `<marquee>`. Ella ha estado persiguiendo ese alto desde entonces. Cuando no está agregando errores en React, disfruta lanzando memes descarados en el chat y jugando demasiados videojuegos con su pareja y su perro Zelda.

Reviewer: ```suggestion
    La carrera de programadora de Lauren alcanzó su punto máximo cuando descubrió la etiqueta `<marquee>`. Ha estado persiguiendo ese momento desde entonces. Cuando no está añadiendo errores a React, disfruta lanzando memes descarados en el chat y jugando a demasiados videojuegos con su pareja y su perra Zelda.
```

</TeamMember>

<TeamMember name="Luna Ruan" permalink="luna-ruan" photo="/images/team/lunaruan.jpg" github="lunaruan" twitter="lunaruan" title="Ingeniera Independiente">

>     Luna aprendió programación porque pensó que significaba crear videojuegos. En cambio, terminó trabajando en la aplicación web de Pinterest y ahora en React. Luna ya no quiere hacer videojuegos, pero planea hacer escritura creativa si alguna vez se aburre.

Reviewer: ```suggestion
    Luna aprendió a programar porque pensaba que significaba crear videojuegos. En lugar de eso, acabó trabajando en la aplicación web Pinterest, y ahora en el propio React. Luna ya no quiere crear videojuegos, pero planea dedicarse a la escritura creativa si alguna vez se aburre.
```

</TeamMember>

<TeamMember name="Luna Wei" permalink="luna-wei" photo="/images/team/luna-wei.jpg" github="lunaleaps" twitter="lunaleaps" title="Ingeniera en Meta">
    Luna aprendió por primera vez los fundamentos de Python de la mano de su padre cuando tenia 6. Desde entonces, ha sido imparable. Luna aspira a ser una generación Z, y el camino hacia el éxito está pavimentado con la defensa del medio ambiente, la jardinería urbana y mucho tiempo de calidad con su Voo-Doo'd (como se muestra en la imagen).
</TeamMember>

<TeamMember name="Matt Carroll" permalink="matt-carroll" photo="/images/team/matt-carroll.png" github="mattcarrollcode" twitter="mattcarrollcode" title="Defensor del Desarrollador en Meta">

>     Matt se topó con el código y, desde entonces, se enamoró de crear cosas en comunidades que no se pueden crear solo. Antes de React, trabajó en YouTube, Google Assistant, Fuchsia, Google Cloud AI y Evernote. Cuando no está tratando de crear mejores herramientas para desarrolladores, disfruta de las montañas, el jazz y pasar tiempo con su familia.

Reviewer: ```suggestion
    Matt tropezó con la codificación y, desde entonces, se ha enamorado de crear cosas en comunidades que no pueden crearse en solitario. Antes de React, trabajó en YouTube, Google Assistant, Fuchsia, Google Cloud AI y Evernote. Cuando no está intentando crear mejores herramientas para desarrolladores, le gusta la montaña, el jazz y pasar tiempo con su familia.
```

</TeamMember>

<TeamMember name="Mengdi Chen" permalink="mengdi-chen" photo="/images/team/mengdi-chen.jpg" github="mondaychen" twitter="mengdi_en" title="Ingeniero en Meta">

>     Mientras trabajaba en su título de Artes digitales, Mengdi se jactaba de sus habilidades de front-end porque su CSS funcionaba perfectamente incluso en IE6. Pero pronto React le abrió una nueva puerta de programación, y desde entonces ha estado soñando con unirse al equipo de React. Fuera del trabajo, suele estar ocupado persiguiendo a sus dos hijos o coleccionando recetas extrañas.

Reviewer: ```suggestion
    Mientras cursaba la carrera de Artes Digitales, Mengdi presumía de sus habilidades front-end porque su CSS funcionaba perfectamente incluso en IE6. Pero pronto React le abrió una nueva puerta de la programación, y desde entonces sueña con unirse al equipo de React. Fuera del trabajo, suele estar ocupado persiguiendo a sus dos hijos o coleccionando recetas extrañas.
```

</TeamMember>

<TeamMember name="Mofei Zhang" permalink="mofei-zhang" photo="/images/team/mofei-zhang.png" github="mofeiZ" title="Ingeniera en Meta">

>     Mofei comenzó a programar cuando se dio cuenta de que podía ayudarla a hacer trampa en los videojuegos. Se centró en los sistemas operativos en la escuela de pregrado / posgrado, pero ahora se encuentra felizmente jugando con React. Fuera del trabajo, le gusta depurar problemas de boulder y planificar su(s) próximo(s) viaje(s) como mochilera.

Reviewer: ```suggestion
    Mofei empezó a programar cuando se dio cuenta de que podía ayudarle a hacer trampas en los videojuegos. Se centró en los sistemas operativos en la licenciatura / posgrado, pero ahora se encuentra feliz jugando en React. Fuera del trabajo, disfruta depurando problemas de boulder y planificando su(s) próximo(s) viaje(s) de mochilera.
```

</TeamMember>

<TeamMember name="Rick Hanlon" permalink="rick-hanlon" photo="/images/team/rickhanlonii.jpg" github="rickhanlonii" twitter="rickhanlonii" personal="rickhanlon.codes" title="Ingeniero en Meta">

>     Ricky se especializó en matemáticas teóricas y de alguna manera se encontró en el equipo de React Native durante un par de años antes de unirse al equipo de React. Cuando no está programando, puedes encontrarlo haciendo snowboard, andando en bicicleta, escalando, jugando golf o cerrando problemas de GitHub que no coinciden con la plantilla de problemas.

Reviewer: ```suggestion
    Ricky se especializó en matemáticas teóricas y de alguna manera se encontró en el equipo de React Native durante un par de años antes de unirse al equipo de React. Cuando no está programando, puedes encontrarlo haciendo snowboard, andando en bicicleta, escalando, jugando al golf o cerrando problemas de GitHub que no coinciden con la plantilla de problemas.
```

</TeamMember>

<TeamMember name="Samuel Susla" permalink="samuel-susla" photo="/images/team/sam.jpg" github="sammy-SC" twitter="SamuelSusla" title="Ingeniero en Meta">

>     El interés de Samuel por la programación comenzó con la película Matrix. Todavía tiene el protector de pantalla de Matrix. Antes de trabajar en React, se centró en escribir aplicaciones para iOS. Fuera del trabajo, a Samuel le gusta jugar voleibol de playa, squash, bádminton y pasar tiempo con su familia.

Reviewer: ```suggestion
    El interés de Samuel por la programación empezó con la película Matrix. Todavía tiene el protector de pantalla de Matrix. Antes de trabajar en React, se dedicaba a escribir aplicaciones para iOS. Fuera del trabajo, Samuel disfruta jugando al voley de playa, squash, bádminton y pasar tiempo con su familia.
```

</TeamMember>

<TeamMember name="Sathya Gunasekaran " permalink="sathya-gunasekaran" photo="/images/team/sathya.jpg" github="gsathya" twitter="_gsathya" title="Ingeniero en Meta">

>     Sathya odiaba el Libro del dragón en la escuela, pero de alguna manera terminó trabajando en compiladores durante toda su carrera. Cuando no está compilando componentes de React, está bebiendo café o comiendo otro Dosa.

Reviewer: ```suggestion
    Sathya odiaba el Libro del Dragón en la escuela, pero de alguna manera terminó trabajando en compiladores toda su carrera. Cuando no está compilando componentes de React, está bebiendo café o comiendo otra Dosa.
```

</TeamMember>

<TeamMember name="Sean Keegan" permalink="sean-keegan" photo="/images/team/sean-keegan.jpg" github="seanryankeegan" twitter="DevRelSean" title="Defensor del Desarrollador en Meta">

>     Después de una primera carrera como profesor de matemáticas, Sean recordó que una clase de introducción a la ciencia informática que tenía que tomar como requisito previo y pensó "¡eso fue algo divertido!". Después de un bootcamp de programación y varios trabajos tecnológicos, Sean descubrió la defensa de los desarrolladores y no ha mirado hacia atrás. Fuera del trabajo, Sean disfruta del disco volador, los videojuegos y de investigar (pero rara vez implementar) mejores formas de cuidar sus plantas de interior.

Reviewer: ```suggestion
    Tras una primera carrera como profesor de matemáticas, Sean recordó aquella clase de introducción a las ciencias de la computación que tuvo que tomar como prerrequisito y pensó "¡eso era divertido!". Después de un curso de programación y varios trabajos tecnológicos, Sean descubrió la defensa de los desarrolladores y no ha mirado atrás. Fuera del trabajo, a Sean le gusta jugar al frisbee, los videojuegos e investigar (pero rara vez poner en práctica) mejores formas de cuidar sus plantas de interior.
```

</TeamMember>

<TeamMember name="Sebastian Markbåge" permalink="sebastian-markbåge" photo="/images/team/sebmarkbage.jpg" github="sebmarkbage" twitter="sebmarkbage" title="Ingeniero en Vercel">

>     Sebastián se especializó en psicología. Suele estar callado. Incluso cuando dice algo, a menudo no tiene sentido para el resto de nosotros hasta unos meses después. La forma correcta de pronunciar su apellido es "mark-boa-geh", pero se conformó con "mark-beige" por pragmatismo, y así es como aborda React.

Reviewer: ```suggestion
    Sebastian se especializó en psicología. Suele ser callado. Incluso cuando dice algo, a menudo no tiene sentido para el resto de nosotros hasta unos meses después. La forma correcta de pronunciar su apellido es "mark-boa-geh", pero se ha decantado por "mark-beige" por pragmatismo, y así es como se acerca a React.
```

</TeamMember>

<TeamMember name="Sebastian Silbermann" permalink="sebastian-silbermann" photo="/images/team/sebsilbermann.jpg" github="eps1lon" twitter="sebsilbermann" title="Ingeniero Independiente">

>     Sebastian aprendió a programar para hacer que los juegos de navegador que jugaba durante la clase fueran más divertidos. Eventualmente, esto lo llevó a contribuir a la mayor cantidad posible de código fuente abierto. Fuera de la programación, está ocupado asegurándose de que la gente no lo confunda con los otros Sebastians y Zilberman de la comunidad React.

Reviewer: ```suggestion
    Sebastian aprendió a programar para hacer más divertidos los juegos de navegador a los que jugaba en clase. Con el tiempo, esto le llevó a contribuir a la mayor cantidad de código abierto posible. Fuera de la programación, está ocupado asegurándose de que la gente no lo confunda con los otros Sebastian y Zilberman de la comunidad React.
```

</TeamMember>

<TeamMember name="Seth Webster" permalink="seth-webster" photo="/images/team/seth.jpg" github="sethwebster" twitter="sethwebster" personal="sethwebster.com" title="Gerente de ingeniería en Meta">

>     Seth comenzó a programar cuando era niño y crecía en Tucson, AZ. Después de la escuela, le picó el gusanillo de la música y estuvo de gira durante unos 10 años antes de volver al *trabajo*, comenzando con Intuit. En su tiempo libre, le encanta [tomar fotografías](https://www.sethwebster.com) y volar para rescatar animales en el noreste de los Estados Unidos.

Reviewer: ```suggestion
    Seth comenzó a programar cuando era niño y crecía en Tucson, AZ. Después de la escuela, le picó el gusanillo de la música y estuvo de gira durante unos 10 años antes de volver al *trabajo*, comenzando con Intuit. En su tiempo libre, le encanta [tomar fotografías](https://www.sethwebster.com) y volar para rescatar animales en el noreste de los Estados Unidos.
```
```suggestion
    Seth empezó a programar de niño en Tucson, Arizona. Después de la escuela, le picó el gusanillo de la música y fue músico de gira durante unos 10 años antes de volver al *trabajo*, comenzando con Intuit. En su tiempo libre, le encanta [hacer fotos](https://www.sethwebster.com) y volar para rescatar animales en el noreste de Estados Unidos.
```

</TeamMember>

<TeamMember name="Sophie Alpert" permalink="sophie-alpert" photo="/images/team/sophiebits.jpg" github="sophiebits" twitter="sophiebits" personal="sophiebits.com" title="Ingeniera Independiente">

>     Cuatro días después del lanzamiento de React, Sophie reescribió la totalidad de su proyecto actual para usarlo, y ahora se da cuenta de que quizás fue un poco imprudente. Después de convertirse en la responsable principal del proyecto, se preguntó por qué Facebook no le pagaba como a todos los demás y se unió oficialmente al equipo para liderar React durante sus años de adolescencia. Aunque renunció a ese trabajo hace años, de alguna manera todavía está en los chats grupales del equipo y "aportando valor".

Reviewer: ```suggestion
    Cuatro días después del lanzamiento de React, Sophie reescribió la totalidad de su proyecto actual para utilizarlo, lo que ahora se da cuenta de que quizás fue un poco imprudente. Después de convertirse en la principal responsable del proyecto, se preguntó por qué Facebook no le pagaba como a los demás y se unió oficialmente al equipo para dirigir React durante sus años de adolescencia. Aunque dejó el trabajo hace años, sigue participando en los chats de grupo del equipo y "aportando valor".
```

</TeamMember>

<TeamMember name="Tianyu Yao" permalink="tianyu-yao" photo="/images/team/tianyu.jpg" github="tyao1" twitter="tianyu0" title="Ingeniero en Meta">

>     El interés de Tianyu por las computadoras comenzó cuando era niño porque le encantan los videojuegos. Así que se especializó en informática y todavía juega juegos infantiles como League of Legends. Cuando no está frente a una computadora, disfruta jugar con sus dos gatitos, hacer caminatas y andar en kayak.

Reviewer: ```suggestion
    El interés de Tianyu por la informática empezó de niño porque le encantan los videojuegos. Por eso se especializó en informática y sigue jugando a juegos infantiles como League of Legends. Cuando no está delante de una computadora, le gusta jugar con sus dos gatitos, hacer senderismo y navegar en kayak.
```

>     El interés de Tianyu por las computadoras comenzó cuando era niño porque le encantan los videojuegos. Así que se especializó en informática y todavía juega juegos infantiles como League of Legends. Cuando no está frente a una computadora, disfruta jugar con sus dos gatitos, hacer caminatas y andar en kayak.

Reviewer: ```suggestion
    El interés de Tianyu por las computadoras empezó de niño porque le encantan los videojuegos. Por eso se especializó en informática y sigue jugando a juegos infantiles como League of Legends. Cuando no está delante de una computadora, le gusta jugar con sus dos gatitos, hacer senderismo y navegar en kayak.
```

</TeamMember>

<TeamMember name="Yuzhi Zheng" permalink="yuzhi-zheng" photo="/images/team/yuzhi.jpg" github="yuzhi" twitter="yuzhiz" title="Gerente de ingeniería en Meta">

>     Yuzhi estudió Ciencias de la Computación en la escuela. Le gustaba la gratificación instantánea de ver cómo el código cobraba vida sin tener que estar físicamente en un laboratorio. Ahora es gerente en la organización React. Antes de la gerencia, solía trabajar en el marco de obtención de datos de Relay. En su tiempo libre, Yuzhi disfruta optimizando su vida a través de proyectos de jardinería y mejoras para el hogar.

Reviewer: ```suggestion
    Yuzhi estudió Ciencias de la Computación en la escuela. Le gustaba la gratificación instantánea de ver cómo el código cobraba vida sin tener que estar físicamente en un laboratorio. Ahora es gerente en React org. Antes de ser gerente, trabajaba en el framework de obtención de datos Relay. En su tiempo libre, Yuzhi disfruta optimizando su vida mediante proyectos de jardinería y mejoras del hogar.
```

</TeamMember>

## Colaboradores anteriores {/*past-contributors*/}

> Puede encontrar a los miembros anteriores del equipo y otras personas que contribuyeron significativamente a React a lo largo de los años en el apartado de [agredecimientos](/community/acknowledgements) page.

Reviewer: ```suggestion
Puedes encontrar a los antiguos miembros del equipo y a otras personas que han contribuido significativamente a React a lo largo de los años en la página de [agradecimientos](/community/acknowledgements).
```
